invenio_audit_logs/services/schema.py

```diff
@@ -34,6 +34,11 @@ class ResourceSchema(Schema):
 class MetadataSchema(Schema):
     """Metadata schema for logging."""
 
+    class Meta:
+        """Meta class to ignore unknown fields."""
+
+        unknown = EXCLUDE  # Ignore unknown fields
```

Comment:
minor: why not RAISE?

Reply:
discussed in person, we want to filter out any unwanted metadata keys